src/resources/js/App.vue

```diff
@@ -4,6 +4,7 @@
             <div class="collapse navbar-collapse">
                 <div class="navbar-nav">
                     <router-link to="/hello" class="nav-item nav-link">Home</router-link>
+                    <router-link to="/register" class="nav-item nav-link">Register</router-link>
```

Comment:
even if this is only a temporary file until we've got the website completely routed, the link text should be in french as it can be seen by the user (line 7)

Reply:
Changed in 313adfc